cmd/flowrunner/main.go

```diff
@@ -21,8 +21,8 @@ import (
 )
 
 type Output struct {
-	Session json.RawMessage        `json:"session"`
-	Events  []*utils.TypedEnvelope `json:"events"`
+	Session  json.RawMessage   `json:"session"`
+	EventLog []json.RawMessage `json:"events"`
```

Comment:
Haven't done it yet but I think this should now be something other than just `events` for clarity - e.g. `event_log` or `log`